api_hammer.gemspec

```diff
@@ -34,6 +34,4 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest-reporters'
   spec.add_development_dependency 'yard'
   spec.add_development_dependency 'simplecov'
-  spec.add_development_dependency 'activesupport'
```

Comment:
oh wait, it isn't here ... that was a different branch. I do need activesupport though.